lib/t/matches.ex

```diff
@@ -453,14 +479,24 @@ defmodule T.Matches do
 
     Multi.new()
     |> Multi.run(:unmatch, fn _repo, _changes ->
-      Match
-      |> where(user_id_1: ^uid1)
-      |> where(user_id_2: ^uid2)
-      |> select([m], %{id: m.id, users: [m.user_id_1, m.user_id_2]})
-      |> Repo.delete_all()
-      |> case do
-        {1, [match]} -> {:ok, match}
-        {0, _} -> {:error, :match_not_found}
+      match_id =
+        Match
+        |> where(user_id_1: ^uid1)
+        |> where(user_id_2: ^uid2)
+        |> select([m], m.id)
+        |> Repo.one()
+
+      if match_id do
+        :ok = T.Calls.voicemail_delete_all(match_id)
```

Comment:
There's a bit of repetition in deleting voicemail on unmatch, but I've checked and it works. I'll refactor it later.